### ISSUE TITLE
Schizencephaly type 1 

### ISSUE THREAD

**Issue by Suz13** [obophenotype/human-phenotype-ontology#6991]
**Preferred term label:**
Schizencephaly (type 1)

**Synonyms**

**Definition (free text, please give PubMed ID)**
Trans-mantle column of abnormal grey matter but no evidence of a CSF-containing cleft on MR imaging  

Of 11 fetuses reviewed, 0/11 had Schizencephaly (type 1) (v 2/21 paediatric cases)
PMID: 30027296

Comment: could be further subdivided into unilateral and bilateral

**Parent term (use hpo.jax.org/app)**
HP:0010636

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 269160
ORPHA:799 Schizencephaly
ORPHA:485275 Acquired schizencephaly
ORPHA:481986 Familial schizencephaly

**Your nano-attribution (ORCID)**
0000-0001-7915-6072

**Comment by Suz13** [comment]
#prenatal/perinatal/neonatal

**Comment by patt1morgan** [comment]
@carlotarodo, @TonyRoscioli, @Suz13.   Thank you for agreeing to work with us to advance the perinatal section of the HPO. We would appreciate it if you could review this new term request and approve it or help us improve it. Any suggestions welcome!

**Comment by carlotarodo** [comment]
Hi, not sure what I'm supposed to do.

Schizencephaly: neuronal migration disorder characterized by the presence of clefts that, crossing the cerebral hemisphere, extend from the lateral ventricles to the cortical surface.

Difficult to diagnose in prenatal setting, often missed in routine scan.

Approve it.

**Comment by pnrobinson** [comment]
@Suz13 
Here are proposed definitions -- do you agree?

Schizencephaly (type 1)—A type of Schizencephaly in which there is a trans-mantle column of abnormal grey matter but no evidence of a CSF-containing cleft on MR imaging.
Schizencephaly (type 2)—A type of Schizencephaly in which CSF-containing cleft is present with abutting lining lips of abnormal grey matter.
Schizencephaly (type 3)—A type of Schizencephaly in which CSF-containing cleft present with non-abutting lining lips of abnormal grey matter.

PMID:30027296. (see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6096842/)

**Comment by pnrobinson** [comment]
adding PMID:3111205 for type I Schizencephaly

**Comment by patt1morgan** [comment]
Hi Peter
Here was another Pub med article with a good description of the types of schizencephaly, including the terms 'open lip and closed lip which you often see.
 PMID:32809748

Type 1 (trans-mantle): No CSF-containing cleft on magnetic resonance imaging (MRI), but contains a trans-mantle column of abnormal gray matter.

Type 2 (closed-lip): Presence of cleft containing CSF, but the lining lips of abnormal gray matter are abutting and opposed to each other. Milder form, can be asymptomatic or diagnosed in adult patients.
Type 3 (open-lip): Presence of cleft containing CSF. The lining lips of abnormal gray matter are not abutting each other.  Severe form of schizencephaly that can manifest by refractory epilepsy, intellectual disability, varying degrees of paralysis from hemiparesis to quadriparesis.

**Comment by Suz13** [comment]
^ Description from @patt1morgan is clear

**Comment by pnrobinson** [comment]
Thanks @patt1morgan and @Suz13 , I am adding these terms
